portlets/src/main/webapp/vue-app/badgesOverview/components/BadgesOverviewItem.vue

```diff
@@ -15,23 +15,27 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <div class="px-2">
-    <v-avatar
-      class="d-flex flex-column BadgeItemAvatar content-box-sizing clickable mx-auto"
-      :title="badge.badgeLabel"
-      :tile="!skeleton"
-      @click="openDrawer">
-      <img
-        :class="skeleton && 'skeleton-background no-border'"
-        :src="badge.avatar">
-    </v-avatar>
-    <div
-      :class="skeleton && 'skeleton-text skeleton-background skeleton-text-height skeleton-border-radius px-6'"
-      class="d-block text-center mt-2 clickable"
-      @click="openDrawer">
-      {{ skeleton && '&nbsp;' || badge.domainLabel }}
-    </div>
-  </div>
+  <v-tooltip bottom>
+    <template #activator="{ on }">
+      <div class="px-2" v-on="on">
+        <v-avatar
+          class="d-flex flex-column BadgeItemAvatar content-box-sizing clickable mx-auto"
+          :tile="!skeleton"
+          @click="openDrawer">
+          <img
+            :class="skeleton && 'skeleton-background no-border'"
+            :src="badge.avatar">
+        </v-avatar>
+        <div
+          :class="skeleton && 'skeleton-text skeleton-background skeleton-text-height skeleton-border-radius px-6'"
```

Comment:
We don't use `skeleton` attribute IMO since we don't display it. Could you please delete this attribute and its usage.

Reply:
done!